applications/matter_weather_station/README.rst

```diff
@@ -27,9 +27,12 @@ The application supports the following development kits:
 To commission the weather station device and control it remotely through a Thread network, you also need a Matter controller device :ref:`configured on PC or smartphone <ug_matter_configuring>` (which requires additional hardware depending on which setup you choose).
 The recommended way of getting measurement values is using the mobile Matter controller application that comes with a neat graphical interface, performs measurements automatically and visualizes the data.
 
-To program the Thingy:53 device, you need the external J-Link programmer.
+To program a Thingy:53 device that has no bootloader, you need the external J-Link programmer.
```

Comment:
To program a Thingy:53 device where the preprogrammed bootloader has been erased, you need the external J-Link programmer

Reply:
Changed